### DISCUSSION TITLE
Cheetah sampler help

### DISCUSSION THREAD

**Discussion by berni4** [keirf/flashfloppy#951] [Q&A]
Hello, I am working on a Cheetah SX16 sampler and am trying FlashFloppy for the first time. The Cheetah sampler device is not supported by HxC (no files in QuickInstall_FloppyDiskImages). I do have some disks formatted for it, and I used a USB floppy drive and used dd to create images from them. I then tried to make HxC files using the HxCFloppyEmulator software.

When the files are in .img format, I can read the file names on the Cheetah sampler, but I can't load them. The same thing happens with the HxC files when I have indexed mode enabled in the ff.cfg file. The Cheetah uses S0, which is also the only jumper I have set.

Quick question: I’m a bit concerned about using a USB floppy drive to read the 720k floppies, maybe a internal drive would be better. Do you think that, since I can read the file names, the .img files were “ripped” correctly by dd?

Also, what settings would you suggest for the ff.cfg file?

I have added the raw .img i got from dd
[floppy3.zip](https://github.com/user-attachments/files/17757856/floppy3.zip)

**Comment by keirf** [comment]
You could also try with and without a jumper at JC. The rip is probably okay, I think you'd get an error if the sector layout wasn't 720k. It's possible the sectors need to be interleaved though. The best way to check that would be to get a raw flux dump using a device such as Greaseweazle, and upload that flux image file for inspection. Then we could see the true low-level layout of a disk formatted for the Cheetah.
